src/test/config.sol

```diff
@@ -1669,5 +1669,65 @@ contract Config {
             calc_cut:     9900,
             lerp:         false
         });
+        afterSpell.collaterals["RWA008-A"] = CollateralValues({
+            aL_enabled:   false,         // DssAutoLine is enabled?
+            aL_line:      0,             // In whole Dai units
+            aL_gap:       0,             // In whole Dai units
+            aL_ttl:       1 hours,       // In seconds
```

Comment:
Check previous patterns and adjust parameters accordingly (e.g. `RWA006`)

Reply:
Fixed